src/Core/Core.csproj

```diff
@@ -62,5 +62,6 @@
   <ItemGroup>
     <Folder Include="Resources\" />
     <Folder Include="Properties\" />
+    <Folder Include="Utilities\Plan\" />
```

Comment:
Is this needed? I'm not sure, I'm guessing maybe Rider just added it there but it's not actually required?